components/odh-notebook-controller/e2e/helper_test.go

```diff
@@ -3,18 +3,20 @@ package e2e
 import (
 	"crypto/tls"
 	"fmt"
-	netv1 "k8s.io/api/networking/v1"
 	"log"
 	"net/http"
 	"time"
 
+	netv1 "k8s.io/api/networking/v1"
+
```

Comment:
why this empty space? I'd delete and let gofmt to reshuffle it into the block below

Reply:
updated, hopefully for your happiness, Jiri :grinning:

Reply:
gofmt is not like black/the-other-python-thing-we-now-have-in-ods-ci, it does take hints from what you've originally done in the file, regarding some things

black completely parses and spits out from AST, so it does not take hints